.github/workflows/crypto-test-harness.yml

```diff
@@ -43,3 +43,40 @@ jobs:
         with:
           name: tcrypt_job
           path: ./tcrypt.out
+  cfgcheck:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y bc libelf-dev pahole
+          mkdir -p out
```

Comment:
Hmm. Does this need to be AFTER the actions/checkout@v3 step? All of the steps failed because of:

```
/home/runner/work/_temp/1f94bbb9-a7e7-4490-a97b-3938df7b1511.sh: line 6: out/config-integration.out: No such file or directory
Error: Process completed with exit code 1.
```